File: src/main/java/com/rs/java/game/area/banks/FaladorWestBank.java
package com.rs.java.game.area.banks;

import com.rs.java.game.WorldTile;
import com.rs.java.game.area.Area;
import com.rs.java.game.area.Shape;
import com.rs.java.game.area.shapes.Polygon;

public final class FaladorWestBank extends Area {

	@Override
	public Area update() {
		return this;
	}

	@Override
	public String name() {
		return "Falador West Bank";
	}

	public FaladorWestBank() {
		super(
				new Polygon(new WorldTile[] {
				new WorldTile(2943, 3368, 0),
				new WorldTile(2943, 3373, 0),
				new WorldTile(2947, 3373, 0),
				new WorldTile(2947, 3369, 0),
				new WorldTile(2949, 3369, 0),
				new WorldTile(2949, 3368, 0),
				new WorldTile(2943, 3368, 0),
		}) );
	}

	@Override
	public boolean member() {
		return false;
	}

	@Override
	public Environment environment() {
		return Environment.SAFEZONE;
	}

}

File: src/main/java/com/rs/kotlin/game/world/area/areas/banks/FaladorWestBank.kt
package com.rs.kotlin.game.world.area.areas.banks

import com.rs.java.game.WorldTile
import com.rs.kotlin.game.world.area.Area
import com.rs.kotlin.game.world.area.shape.Polygon

class FaladorWestBank : Area(
    Polygon(
        arrayOf(
            WorldTile(2943, 3368, 0),
            WorldTile(2943, 3373, 0),
            WorldTile(2947, 3373, 0),
            WorldTile(2947, 3369, 0),
            WorldTile(2949, 3369, 0),
            WorldTile(2949, 3368, 0),
            WorldTile(2943, 3368, 0),
        )
    )
) {
    override fun update(): Area {
        return this
    }

    override fun name(): String {
        return "Falador West Bank"
    }

    override fun member(): Boolean {
        return false
    }

    override fun environment(): Environment {
        return Environment.SAFEZONE
    }
}
